pkg/apis/kai/v1/global.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2025 NVIDIA CORPORATION
+// SPDX-License-Identifier: Apache-2.0
+
+package v1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/utils/ptr"
+
+	"github.com/NVIDIA/KAI-scheduler/pkg/apis/kai/v1/common"
+)
+
+// GlobalConfig defines the global configuration of the system
+type GlobalConfig struct {
+	// Openshift configures the operator to install on Openshift
+	// +kubebuilder:validation:Optional
+	Openshift *bool `json:"openshift,omitempty"`
+
+	// Affinity defined affinity to the all microservices
```

Comment:
```suggestion
	// Affinity defines affinity for kai system services
```
To avoid confusion with maybe default affinity for jobs and stuff like that